### DISCUSSION TITLE
Heater termostat Tuya TS0601 By _TZE200_hue3yfsn

### DISCUSSION THREAD

**Discussion by Hewelius** [Koenkk/zigbee2mqtt#10031] [Devices]
Dear colleagues programmers,
I'm trying to add to my system Tuya heater thermostat TS0601 and manufacturer code _TZE200_hue3yfsn. Generally it looks to be supported. I have checked in tuya.js. But I'm still getting info from z2m that it is not supported. I have been trying to add local converter but other problem appears: e.window_open is not a function.
Sorry for screenshot in polish.

![Screenshot_20211205-211221062](https://user-images.githubusercontent.com/93792752/144762303-0c6395aa-2c3b-446b-88d3-a216cecd67d0.jpg)

**Comment by jdnowak** [comment]
You need to move dev branch. 
P.S. If you use ha, Easiest way is to install ziggbee2mqtt edge version.
